src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,91 @@
+package parser;
+
+import customer.Customer;
+import exceptions.CustomerException;
+import java.util.Scanner;
+
+public class Parser {
+
+    public static Scanner scanner = new Scanner(System.in);
+    public static String userInput;
+    private static final String addCustomerCommand = "add-user";
+
+    /**
+     * Gets the first word of user input.
+     */
+    public static String getAction(){
+        System.out.println("What would you like to do?");
+        System.out.println("_".repeat(60));
+        String userInput = scanner.nextLine();
+        userInput = userInput.trim();
+        String[] words = userInput.split(" ");
+        Parser.userInput = userInput;
+        return words[0];
+    }
+
+    /**
+     * Creates new customer object based on user input.
+     *
+     * @throws CustomerException if input is not compliant with format.
+     * @throws NumberFormatException if the age and contact content are not integer string.
+     */
+    public static Customer parseIntoCustomer() throws CustomerException , NumberFormatException {
+        userInput = userInput.substring(addCustomerCommand.length()).trim();
+        String[] parameters = { "/u" , "/a" , "/c"};
+        String[] parameterContents;
+        if(isValidSequence(parameters)){
```

Comment:
Maybe you can follow the "make the happy path prominent" coding standard here?? (i.e. Use the guard clause "if( !isValidSequence(parameters) ), then throw the exception here in the if block.